cmd/kubeadm/app/phases/addons/proxy/manifests.go

```diff
@@ -48,6 +48,40 @@ data:
         tokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
 `
 
+	// KubeProxyConfigMap is the proxy ConfigMap manifest
+	KubeProxyConfigMap19 = `
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: kube-proxy
+  namespace: kube-system
+  labels:
+    app: kube-proxy
+data:
+  kubeconfig.conf: |-
+    apiVersion: v1
+    kind: Config
+    clusters:
+    - cluster:
+        certificate-authority: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+        server: {{ .MasterEndpoint }}
+      name: default
+    contexts:
+    - context:
+        cluster: default
+        namespace: default
+        user: default
+      name: default
+    current-context: default
+    users:
+    - name: default
+      user:
+        tokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+  config.conf: |-
+    apiVersion: kubeproxyconfig/v1alpha1
```

Comment:
?
Isn't this automatically populated in ProxyConfig? I would expect it to

Reply:
This is what the populated manifest looks like  

config.conf: |-
    apiVersion: kubeproxyconfig/v1alpha1
    kind: KubeProxyConfiguration
    BindAddress: 0.0.0.0
    ClientConnection:
      AcceptContentTypes: ""
      Burst: 0
      ContentType: ""
      KubeConfigFile: ""
      QPS: 0
    ClusterCIDR: ""
    ConfigSyncPeriod: 0s
    Conntrack:
      Max: null
      MaxPerCore: null
      Min: null
      TCPCloseWaitTimeout: null
      TCPEstablishedTimeout: null
    EnableProfiling: false
    FeatureGates: ""
    HealthzBindAddress: 0.0.0.0:10256
    HostnameOverride: ""
    IPTables:
      MasqueradeAll: false
      MasqueradeBit: null
      MinSyncPeriod: 0s
      SyncPeriod: 0s
    IPVS:
      MinSyncPeriod: 0s
      Scheduler: ""
      SyncPeriod: 0s
    MetricsBindAddress: 127.0.0.1:10249
    Mode: ""
    OOMScoreAdj: null
    PortRange: ""
    ResourceContainer: ""
    UDPIdleTimeout: 0s

Reply:
please remove `kind` and `apiVersion` here, should not be needed when using the external version